public/js/utils/source-map.js

```diff
@@ -89,17 +89,22 @@ async function getOriginalLocation(state: AppState, location: Location) {
     return location;
   }
 
-  const _isGenerated = await isGenerated(source.toJS());
-
-  if (_isGenerated) {
+  if (isGenerated(source.toJS())) {
     const originalPosition = await getOriginalSourcePosition(
       source.toJS(),
       location
     );
 
     const { url, line } = originalPosition;
+    if (!url) {
+      return {
+        sourceId: source.get("id"),
+        line: location.line
+      };
+    }
```

Comment:
This is an interesting case. The frame locations have undefiend columns,
which is not sufficient for finding the original location in some cases like `  bar()`

```
bar = function() {
var x = 7;
debugger; }
 bar();
```